pallets/vc-management/src/lib.rs

```diff
@@ -218,19 +239,45 @@ pub mod pallet {
 
 	#[pallet::call]
 	impl<T: Config> Pallet<T> {
+		/// add an account to the delegatees
 		#[pallet::call_index(0)]
+		#[pallet::weight(<T as Config>::WeightInfo::add_delegatee())]
+		pub fn add_delegatee(origin: OriginFor<T>, account: T::AccountId) -> DispatchResult {
+			let _ = T::DelegateeAdminOrigin::ensure_origin(origin)?;
+			// we don't care if `account` already exists
+			Delegatee::<T>::insert(account.clone(), ());
+			Self::deposit_event(Event::DelegateeAdded { account });
+			Ok(())
+		}
+
+		/// remove an account from the delegatees
+		#[pallet::call_index(1)]
+		#[pallet::weight(<T as Config>::WeightInfo::remove_delegatee())]
+		pub fn remove_delegatee(origin: OriginFor<T>, account: T::AccountId) -> DispatchResult {
+			let _ = T::DelegateeAdminOrigin::ensure_origin(origin)?;
+			ensure!(Delegatee::<T>::contains_key(&account), Error::<T>::DelegateeNotExist);
+			Delegatee::<T>::remove(account.clone());
+			Self::deposit_event(Event::DelegateeRemoved { account });
+			Ok(())
+		}
+
+		#[pallet::call_index(2)]
```

Comment:
Q: does this change in `call_index` affect the encoding? More to the point, is this a breaking change?

Reply:
call indices change is a breaking change

Reply:
It's a breaking change in the sense that you can't use the cached index/metadata to construct the offline extrinsic. Let me bump the runtime version and update the parachain metadata to be more explicit.